api/desecapi/management/commands/outreach-email.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import sys
+
+from django.core.exceptions import ImproperlyConfigured
+from django.core.management import BaseCommand
+from django.template import engines
+from django.template.backends.django import DjangoTemplates
+from django.urls import resolve, reverse
+
+from desecapi.models import User
+
+
+def _get_default_template_backend():
+    # Ad-hoc implementation of https://github.com/django/django/pull/15944
+    for backend in engines.all():
+        if isinstance(backend, DjangoTemplates):
+            return backend
+    raise ImproperlyConfigured("No DjangoTemplates backend is configured.")
+
+
+class Command(BaseCommand):
+    help = 'Reach out to users with an email. Takes email template on stdin.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('email', nargs='*', help='User(s) to contact, identified by their email addresses')
```

Comment:
Add to help: Defaults to everyone with outreach preference

Reply:
done